### PR TITLE
nrf_rpc: Add changelog file.

### PR DESCRIPTION
**Pull request by maje-emb**
This adds CHANGELOG.rst file.
The file contains the changes that have been made
since the library was released.

### CODE REVIEW

**Comment by peknis** [comment]
You also need to add this file to this page https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nrf_rpc/README.html. Otherwise, it will not be shown.

File: nrf_rpc/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Main branch
+***********
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+
+* Added configurations number of ZCBOR backups.
```
Comment (peknis) [inline nrf_rpc/CHANGELOG.rst:20]: A bit unclear what is "configurations number", needs to be clarified. ZCBOR -> zcbor.

File: nrf_rpc/CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+nRF Connect SDK v1.6.0
+**********************
+
+All the notable changes added to the |NCS| v1.6.0 release are documented in this section.
+
+Changes
+=======
+
+* Renamed nrf_ernno.h to nrf_rpc_ernno.h.
```
Comment (peknis) [inline nrf_rpc/CHANGELOG.rst:83]:
```suggestion
* Renamed :file:`nrf_ernno.h` to :file:`nrf_rpc_ernno.h`.
```

File: nrf_rpc/CHANGELOG.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+nRF Connect SDK v1.7.0
+**********************
+
+All the notable changes added to the |NCS| v1.7.0 release are documented in this section.
+
+Changes
+=======
+
+* Used :kconfig: role as a reference to the Kconfig options.
```
Comment (peknis) [inline nrf_rpc/CHANGELOG.rst:73]: We use :kconfig:option:`CONFIG_OPTION_NAME` for the formatting, but I could not find any instances within the RPC docs in nrfxlib.

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by grochu — APPROVED** [review]
Thanks for adding this file and therefore restoring the history of this module - it's important to merge this and record there all changes in the nRF RPC from now on.

File: nrf_rpc/CHANGELOG.rst (hunk 1 above)
Comment (maje-emb) [reply nrf_rpc/CHANGELOG.rst:20]: ACK. Changed to: _Added :kconfig:option:CONFIG_NRF_RPC_ZCBOR_BACKUPS to configure number of zcbor backups._

File: nrf_rpc/CHANGELOG.rst (hunk 3 above)
Comment (maje-emb) [reply nrf_rpc/CHANGELOG.rst:73]: That's right, this change relates to the template header file.

File: nrf_rpc/CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Main branch
+***********
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+
+* Added :kconfig:option:CONFIG_NRF_RPC_ZCBOR_BACKUPS to configure number of zcbor backups.
```
Comment (peknis) [inline nrf_rpc/CHANGELOG.rst:20]:
```suggestion
* Added the :kconfig:option:`CONFIG_NRF_RPC_ZCBOR_BACKUPS` Kconfig option for setting up the number of zcbor backups.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Review by kapi-no — APPROVED** [review]

**Review by peknis — CHANGES_REQUESTED** [review]
Whoops, approved a bit too early. This file still needs to be added to the ncs\nrfxlib\nrf_rpc\README.rst page.

**Comment by maje-emb** [comment]
> Whoops, approved a bit too early. This file still needs to be added to the ncs\nrfxlib\nrf_rpc\README.rst page.

You're right. I totally forgot about that. Fixed.

**Review by peknis — APPROVED** [review]

**Pull request merged**